modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/validation/NameClashModelValidator.java

```diff
@@ -199,12 +220,34 @@ private void validateNameClashes(Collection<? extends Named>... collections) {
             i++;
           }
 
-          errorMessage.append(format(" which name is '%s'. Names should be unique", entry.getKey()));
+          errorMessage.append(format(" which alias name is '%s'. Names should be unique", entry.getKey()));
```

Comment:
why did you added the alias concept in the message? It doesn't always apply and when it does it's kind of obvious...